namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-fala-staging/resources/route53.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_route53_zone" "route53_zone" {
+  name = "staging.find-legal-advice.justice.gov.uk"
```

Comment:
There is a zone exists with name `find-legal-advice.justice.gov.uk`.

 You can use a sub-domain for the existing zone, by creating an ingress using host name "staging.find-legal-advice.justice.gov.uk"

This will avoid creating multiple zones.